src/main/resources/view/MainWindow.fxml

```diff
@@ -7,73 +7,76 @@
 <?import javafx.scene.control.MenuBar?>
 <?import javafx.scene.control.MenuItem?>
 <?import javafx.scene.control.SplitPane?>
+<?import javafx.scene.layout.GridPane?>
```

Comment:
Whats the grid pane for by the way?

Reply:
Like display grid, let's us set grid columns of certain percentages